### ISSUE TITLE
When in taskbar segment mode cannot see or use notification area.

### ISSUE THREAD

**Issue by devilsrage** [ChrisAnd1998/TaskbarX#899]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by CrazySaft** [comment]
Yeah I experienced that too would be really good i they could move them also to the middle

**Comment by Cdddo** [comment]
Yes it would be a nice feature to be able to have the tray area also optionally show up when using segment mode...

**Comment by ItsSunnyMonster** [comment]
A few small suggestions: 
1. in the pre-release only the start button is visible. 
2. Start button cannot be pressed in the very bottom left corner. (Many people have a muscle memory to move their mouse to the very bottom left corner of the screen to press the start icon)
